examples/Cargo.lock

```diff
@@ -111,6 +111,12 @@ dependencies = [
  "tokio",
 ]
 
```

Comment:
I don't think we want these changes?

Reply:
No we do not.
But I do not see a way to prevent them.

Reply:
The solution I found was to make the lambda function a template argument to the performance code.